src/main/java/com/store/book/model/Book.java

```diff
@@ -23,30 +28,26 @@
 @SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id = ?")
 @Where(clause = "is_deleted=false")
 public class Book {
-
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-
     @Column(name = "title")
     private String title;
-
     @Column(name = "author")
     private String author;
-
     @Column(name = "isbn", unique = true)
     private String isbn;
-
     @Column(name = "price")
     private BigDecimal price;
-
     @Column(name = "description")
     private String description;
-
     @Column(name = "coverImage")
     private String coverImage;
-
     @Column(name = "is_deleted", nullable = false)
     private Boolean isDeleted = false;
-
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne
```
What relation do Book and User have?